test/stored_queries/test_ncbi_tax.py

```diff
@@ -285,3 +296,4 @@ def test_get_associated_objs(self):
         ids = {ret['ws_obj']['_id'] for ret in results['results']}
         self.assertEqual(assignments, {'assn1', 'assn2'})
         self.assertEqual(ids, {'ws_object_version/1:1:1', 'ws_object_version/1:1:2'})
+        self.assertEqual(results['results'][0]['ws_obj']['type']['type_name'], 'Genome')
```

Comment:
I'd probably check the entire type structure

Reply:
Good call, fixed